deploy.sh

```diff
@@ -10,7 +10,7 @@ export TERRAFORM_ROOT_OPERATIONS="$BITOPS_OPSREPO_ENVIRONMENT_DIR"
 export BITOPS_SCHEMA_ENV_FILE="$TERRAFORM_ROOT_OPERATIONS/ENV_FILE"
 export SCRIPTS_DIR="$TERRAFORM_ROOT_SCRIPTS/scripts"
 
-if [ "$SKIP_DEPLOY_TERRAFORM" == "true" ]; then
+if [ "$SKIP_DEPLOY_TERRAFORM" == "true" ] || [ "$TERRAFORM_SKIP_DEPLOY" == "true" ]; then
```

Comment:
What's the difference between `SKIP_DEPLOY_TERRAFORM` and `TERRAFORM_SKIP_DEPLOY`?